### ISSUE TITLE
Sneaky Data Issues:  Missing Social Links

### ISSUE THREAD

**Issue by GantMan** [infinitered/ChainReactApp2023#137]
e.g. Kadi Kraman doesn't have her correct github/linkedin/website.

I get that the data might not be there yet?   HOWEVER, this makes it very difficult to identify who is and who is not populated with data in the app.    Would be good to not show the icons if the links are not active.  Also, some people might not end up having personal websites etc.   I want to make sure we can catch these in the app. 

TL:DR; hide socials if they aren't populated.

**Comment by yulolimum** [comment]
I was going to open an issue for the social icons not opening anything in the browser on Android - none of the socials for any workshops open anything for me. Not sure if it's just a data issue.

**Comment by yulolimum** [comment]
https://user-images.githubusercontent.com/1775841/233168888-767981b3-2d3a-47d4-8872-2d537c7b040e.mp4
